packages/smart-wallet/src/smartWallet.js

```diff
@@ -371,6 +374,7 @@ export const prepareSmartWallet = (baggage, shared) => {
         address: M.string(),
         bank: M.eref(M.remotable()),
         invitationPurse: PurseShape,
+        // Should not be M.eref, makeRecordedKit assumes resolved
         currentStorageNode: M.eref(StorageNodeShape),
```

Comment:
There should be some kind of runtime change here. This shouldn't be working now. Maybe unused?